webpack.config.dev.js

```diff
@@ -135,19 +131,10 @@ module.exports = {
 };
 
 if (isProduction) {
-  module.exports.plugins.push(
-    new MiniCssExtractPlugin({
-      filename: 'css/[name]-[chunkhash].css',
-    }),
-  );
   module.exports.optimization.minimizer.push(
     new TerserPlugin({
-      include: /\.min\.js$/,
-      extractComments: false,
```

Comment:
the Bundler Plugin already has the [extractComments](https://github.com/ankurk91/vue-toast-notification/pull/98/files#option-extract-comments) option, defaults is false to avoid extraction of files like `*.LICENSE.txt`